.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/metal"]
```

Comment:
Googletest comes for free, but BLT itself would be a submodule. I guess the question is whether we might need to make RAJA-specific changes to anything that couldn't be upstreamed?